backend/api/lsp/commands.go

```diff
@@ -23,4 +23,7 @@ type SetMetadataCommandArguments struct {
 	// For PostgreSQL, it's required.
 	// For other database engines, it's optional.
 	DatabaseName string `json:"databaseName,omitempty"`
+	// The scene is the scene for completion.
+	// Available scenes: "query", "all".
```

Comment:
Add an explanation or example for "query" and "all", plz?

Reply:
Done.